drivers/flash/flash_stm32_ospi.c

```diff
@@ -957,15 +1010,19 @@ static int flash_stm32_ospi_erase(const struct device *dev, off_t addr,
 
 		if (size == dev_cfg->flash_size) {
 			/* Chip erase */
-			LOG_DBG("Chip Erase");
 			cmd_erase.Instruction = (dev_cfg->data_mode == OSPI_OPI_MODE)
 					? SPI_NOR_OCMD_BULKE
 					: SPI_NOR_CMD_BULKE;
 			cmd_erase.AddressMode = HAL_OSPI_ADDRESS_NONE;
-			/* Full chip erase command */
+			/* Full chip erase (Bulk) command */
 			ospi_send_cmd(dev, &cmd_erase);
+			LOG_DBG("Chip Erase");
```

Comment:
The LOG_DBG should be reported the moment you decide the logic goes towards chip erase, not after it has already been invoked.

Reply:
ok, moving above